admin/tests/integration/components/complementary-certifications/attach-form/badges/list_test.js

```diff
@@ -10,12 +10,12 @@ module('Integration | Component | complementary-certifications/attach-badges/lis
   setupIntlRenderingTest(hooks);
   setupMirage(hooks);
 
-  test('[a11y] it should display a message that all inputs are required', async function (assert) {
+  test('[a11y] it should display a message that some inputs are required', async function (assert) {
```

Comment:
Le test fait plus que ça 🤔 
```suggestion
  test('[a11y] it should have required inputs and display a message about them', async function (assert) {
```

Reply:
Le test ne verifie à priori que le message, ca semble donc ok

Reply:
Le test ne fait que constater que la phrase est présente, rien de plus.
Par contre, il y a un test plus bas qui s'assure que chaque champs est obligatoire. 
On peut faire la fusion des deux. L'état actuel me convient bien.